src/utils/analytics.js

```diff
@@ -0,0 +1,25 @@
+import * as Analytics from 'expo-firebase-analytics'
+
+export const publicationSwitchAnalytics = async (from, to) => {
+  console.log('LOGGING EVENT: PUBLICATION SWITCHED')
+  await Analytics.logEvent('PublicationSwitched', {
+    from: from,
+    to: to,
+    purpose: 'user switched the publication',
+  })
+}
+
+export const userViewedArticleAnalytics = async article => {
```

Comment:
it's better here to just pass in each param because `article` itself includes a lot of other irrelevant fields, also need publication here i think